secrethub_wrapper.go

```diff
@@ -0,0 +1,187 @@
+package main
+
+/*
+typedef long long time;
+typedef char* uuid;
+
+struct Secret {
+	uuid SecretID;
+	uuid DirID;
+	uuid RepoID;
+	char* Name;
+	char* BlindName;
+	int VersionCount;
+	int LatestVersion;
+	char* Status;
+	time CreatedAt;
+};
+
+struct SecretVersion {
+	uuid SecretVersionID;
+	struct Secret Secret;
+	int Version;
+	char* Data;
+	time CreatedAt;
+	char* Status;
+};
+#include <stdbool.h>
+*/
+import "C"
+
+import (
+	"encoding/json"
+	"os"
+	"strings"
+
+	"github.com/secrethub/secrethub-go/pkg/secrethub"
+)
+
+// Client creates and returns a new Go client to be used by xgo clients.
+func Client() (*secrethub.Client, error) {
+	options := []secrethub.ClientOption{
+		secrethub.WithAppInfo(&secrethub.AppInfo{
+			Name:    "secrethub-xgo",
+			Version: "0.1.0",
+		}),
+	}
+	client, err := secrethub.NewClient(options...)
```

Comment:
What if the user wants to set any other options? Can we imagine any other way to set them besides environment variables? This is probably very much related to the above question about continuously recreating the client.

Reply:
As mentioned in my other comment, I found a way to make this possible but it will take significant effort (~6h). Do you think we want to have this before the alpha release?

Reply:
I implemented the required changes to make the code forward compatible to adding options.